core/src/main/scala/precog/AutoBump.scala

```diff
@@ -250,25 +281,35 @@ class AutoBump(authorRepository: String, repository: OutdatedRepository, token:
 
   def tryUpdateDependencies: IO[Either[Warnings, (File, String, Option[PullRequestDraft], List[String], ChangeLabel)]] = {
     for {
-      dir <- IO(Files.createTempDirectory("sbt-precog"))
-      dirFile = dir.toFile
-      _ <- Runner[IO](log) ! s"git clone --depth 1 $authenticated ${dirFile.getPath}"
+      path <- IO(Files.createTempDirectory("sbt-precog"))
+      dir = path.toFile
+      _ <- Runner[IO](log).hide(token).stderrToStdout !
+        s"git clone --depth 1 --no-single-branch $authenticated ${dir.getPath}"
```

Comment:
TIL about `--no-single-branch`

Reply:
And it was YIL for me. I don't know when that was introduced -- maybe it was there all along, maybe it was a recent git version. I'm hoping it will be available on the vm.